CredentialCardField.vue

```diff
@@ -60,7 +61,17 @@
         <span>{{name}}</span>
       </div>
       <q-item-section
-        v-if="!sublabels"
+        v-if="component === 'Image'"
+        class="g-field-data-regular">
+        <q-img style="max-width: 128px" class="rounded-borders" :src="value" />
```

Comment:
I don't know if the height should be bounded as well.

Reply:
Yeah, height should be bounded, fixing.